tests/e2e/support/api/keycloak/ocisUserToken.ts

```diff
@@ -0,0 +1,141 @@
+import fetch from 'node-fetch'
+import { TokenEnvironmentFactory } from '../../environment'
+import { config } from '../../../config'
+import { User } from '../../types'
+
+interface ocisTokenForKeycloak {
+  access_token: string
+  refresh_token: string
+}
+
+const authorizationEndpoint = config.keycloakUrl + '/realms/oCIS/protocol/openid-connect/auth'
```

Comment:
```suggestion
const authorizationEndpoint = config.keycloakUrl + '/realms/oCIS/protocol/openid-connect/auth'
```

I found that `authorizationEndpoint` is only used in one function. Is there any reason to make it global rather than local?

Reply:
urls in a global scope of a file is generally ok. because they serve as const variables